lib/consensus/isaac.go

```diff
@@ -225,6 +225,52 @@ func (is *ISAAC) Vote(b ballot.Ballot) (isNew bool, err error) {
 	return
 }
 
+// RemoveOldBallots checkes that `blt` has valid confirmed and proposed time.
```

Comment:
Whyyyyyy do all golang documentation start with the function name ? Presumable having the doc attached to the symbol should be enough ? Especially given the extra long names of Go functions... 

Anyway, you don't have to change it, since it's consistent with the rest. However, "checkes" => "checks"